e2e/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 The OWASP Coraza contributors
+# SPDX-License-Identifier: Apache-2.0
+
+FROM curlimages/curl
```

Comment:
I think dockerfile could have a suffix describing what is this e.g. Dockerfile.curl